website/docs/r/cloud_provider_access.markdown

```diff
@@ -65,4 +75,90 @@ The Cloud Provider Access resource can be imported using project ID and the prov
 $ terraform import mongodbatlas_cloud_provider_access.my_role 1112222b3bf99403840e8934-AWS-5fc17d476f7a33224f5b224e
 ```
 
-See [MongoDB Atlas API](https://docs.atlas.mongodb.com/reference/api/cloud-provider-access-create-one-role/) Documentation for more information.
\ No newline at end of file
+## mongodbatlas_cloud_provider_access (optional)
+
+This is the first resource in the multi-resource path
+
+`mongodbatlas_cloud_provider_access_setup` Allows you to only register AWS IAM roles in Atlas.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+## Example Usage
+
+```hcl
+
+resource "mongodbatlas_cloud_provider_access_setup" "test_role" {
+   project_id = "<PROJECT-ID>"
+   provider_name = "AWS"
+}
+
+```
+
+## Argument Reference
+
+* `project_id` - (Required) The unique ID for the project
+* `provider_name` - (Required) The cloud provider for which to create a new role. Currently only AWS is supported.
+
+
+## Attributes Reference
+
+* `id` - Unique identifier used by terraform for internal management.
+* `aws` - aws related arn roles 
+   * `atlas_assumed_role_external_id` - Unique external ID Atlas uses when assuming the IAM role in your AWS account.
+   * `atlas_aws_account_arn`          - ARN associated with the Atlas AWS account used to assume IAM roles in your AWS account.
+* `created_date`                   - Date on which this role was created.
+* `role_id`                        - Unique ID of this role.
+
+## Import: mongodbatlas_cloud_provider_access_setup
+For consistency is has the same format as the regular mongodbatlas_cloud_provider_access resource 
+can be imported using project ID and the provider name and mongodbatlas role id, in the format 
+`project_id`-`provider_name`-`role_id`, e.g.
+
+```
+$ terraform import mongodbatlas_cloud_provider_access_setup.my_role 1112222b3bf99403840e8934-AWS-5fc17d476f7a33224f5b224e
+```
+
+## mongodbatlas_cloud_provider_authorization (optional)
+
+This is the second resource in the multi-resource path
+
+`mongodbatlas_cloud_provider_access_authorization` Allows you to only authorize an AWS IAM roles in Atlas.
```

Comment:
Slight edit:
 
`mongodbatlas_cloud_provider_access_authorization` Allows you to authorize an AWS IAM roles in Atlas.